parachain/runtime/common/src/benchmarking/orml_vesting.rs

```diff
@@ -0,0 +1,116 @@
+use frame_benchmarking::v2::{account, benchmarks, whitelisted_caller};
+use frame_support::{
+    assert_ok,
+    traits::{Currency, Get},
+};
+use frame_system::RawOrigin;
+use orml_traits::MultiCurrency;
+use orml_vesting::VestingSchedule;
+use primitives::CurrencyId;
+use sp_runtime::traits::StaticLookup;
+use sp_std::prelude::*;
+
+const SEED: u32 = 0;
+
+pub struct Pallet<T: Config>(orml_vesting::Pallet<T>);
+pub trait Config: orml_vesting::Config + currency::Config<CurrencyId = CurrencyId> + loans::Config {}
+pub fn lookup_of_account<T: Config>(
+    who: T::AccountId,
+) -> <<T as frame_system::Config>::Lookup as StaticLookup>::Source {
+    <T as frame_system::Config>::Lookup::unlookup(who)
+}
+
+pub(crate) type BalanceOf<T> =
+    <<T as orml_vesting::Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+pub(crate) type VestingScheduleOf<T> = VestingSchedule<<T as frame_system::Config>::BlockNumber, BalanceOf<T>>;
+
+fn dummy_schedule<T: Config>() -> VestingScheduleOf<T> {
+    VestingSchedule {
+        start: 0u32.into(),
+        period: 10u32.into(),
+        period_count: 2,
+        per_period: 10u32.into(),
+    }
+}
+
+struct Transfer<T: Config> {
+    from: T::AccountId,
+    to: T::AccountId,
+}
+
+fn setup_transfer<T: Config>() -> Transfer<T> {
+    let from: T::AccountId = whitelisted_caller();
+    let to: T::AccountId = account("to", 0, SEED);
+    let amount = 1000u32.into();
+    let native_currency = <T as currency::Config>::GetNativeCurrencyId::get();
+    assert_ok!(orml_tokens::Pallet::<T>::deposit(native_currency, &from, amount));
+
+    Transfer { from, to }
+}
+
+#[benchmarks]
+pub mod benchmarks {
+    use super::{Config, Pallet, *};
+    use orml_vesting::Call;
+
+    #[benchmark]
+    fn claim() {
```

Comment:
Needs to be parametric over the number of schedules: https://github.com/open-web3-stack/open-runtime-module-library/blob/a4977fe70eeea93b072706f7ab2d2577f4a8d320/vesting/src/lib.rs#L248